### ISSUE TITLE
Add Internal DB login

### ISSUE THREAD

**Issue by chriskacerguis** [boozallen/projectjellyfish#1149]
Remove the Auth module, and just doing the login system in core JF.

**Comment by mafernando** [comment]
@chriskacerguis - would it be ok if we setup the login system to enable to you to either login with GitHub or via username/email+password? My concern is that only allowing GitHub logins will prevent JF from being used in environments without GitHub. Off the top of my head, we would have to address what GitHub account to seed the admin user against. 

By allowing both GitHub and username/email+password we preserve the ability to have both GitHub and non-GitHub users in the system and set the stage for enabling authentication with SAML/OAUTH via extensions.

**Comment by chriskacerguis** [comment]
@mafernando Good point, let's go with your suggestions of GH+Username/pass

**Comment by mafernando** [comment]
I went through and verified that we have internal DB logins already working in the current codebase yesterday. 

Since it looks like we're focusing on AD support next week, I went ahead and started looking at https://github.com/boozallen/projectjellyfish/issues/1151 a little. I've setup an account on JumpCloud and am trying to get JF to authenticate against it. 

@chriskacerguis - is there anything more that you wanted from this issue or is it safe to close?

**Comment by chriskacerguis** [comment]
is it a module or internal?

**Comment by mafernando** [comment]
the current jellyfish login system is internal.

**Comment by chriskacerguis** [comment]
have you verified (just wanting to make sure)

**Comment by chriskacerguis** [comment]
I think as long as you have verified it works...then we can just close the ticket.

**Comment by mafernando** [comment]
I went through JF yesterday and verified that Admins can create users from the Users management portal in the Admin section and that a user created that way can login into the system. RSpec passes as well. I looked through the Gemfile and we are not using any custom modules other than `jellyfish-demo`. If there is anything else you'd like me to check, I will. But I think its safe to say this has been verified.
